src/Symfony/Component/Form/Tests/FormFactoryTest.php

```diff
@@ -293,6 +293,15 @@ public function testCreateNamedBuilderThrowsUnderstandableException()
         $this->factory->createNamedBuilder('name', new \stdClass());
     }
 
+    /**
+     * @expectedException        \Symfony\Component\Form\Exception\UnexpectedTypeException
+     * @expectedExceptionMessage Expected argument of type "string, Symfony\Component\Form\ResolvedFormTypeInterface or Symfony\Component\Form\FormTypeInterface", "stdClass" given
+     */
+    public function testCreateBuilderDoNotBreakOnInvalidArgument()
```

Comment:
Maybe the name was ambiguous. "break" referred to the php warning, not to the desirable exception.